### PR TITLE
Fourth Newsletter

### LINKED ISSUE #84: Embedded WG Newsletter #4
Placeholder for the next newsletter, scheduled roughly 2018-04-26

### PR DESCRIPTION
**Pull request by jamesmunns**
Fourth Newsletter

Closes #84

[Rendered Here](https://github.com/rust-lang-nursery/embedded-wg/blob/newsletter-4/newsletters/2018-04-28.md)

### CODE REVIEW

**Comment by jamesmunns** [comment]
Note: @cr1901 will be adding one more project to this newsletter

**Comment by cr1901** [comment]
@jamesmunns and I talked about adding my project to the newsletter, but I forgot to actually send a link :).

And here's a summary:

[AT2XT](https://github.com/cr1901/AT2XT) is open-source hardware that interfaces cheap PS/2 (aka AT) protocol keyboards to vintage IBM PC-class computers expecting an older keyboard protocol (XT). Protocol translation is done using firmware is written in Rust using the msp430 backend and an earlier version of @japaric's RTFM framework.

The firmware is feature-complete; as the Rust compiler evolves, I intend to testing and deploying newly-compiled firmware to ensure AT2XT still works properly. Therefore, AT2XT may also serve as an up-to-date example of how to deploy a Rust application for msp430 family of microcontrollers.

**Comment by jamesmunns** [comment]
Ok to merge now

File: newsletters/2018-04-28.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+If you want to mention something in [the next newsletter], make sure to leave a comment on the issue.
+
+[the next newsletter]: https://github.com/rust-lang-nursery/embedded-wg/issues/93
+[Embedded WG]: https://github.com/rust-lang-nursery/embedded-wg
+
+## Highlights
+
+* The [Embedded WG] IRC meetings are now explicitly public, and the agendas for the next meeting can be found in the [WG Issues], like [this issue] for the next meeting
+* [David] shared his utility [`Form`], which takes a single `lib.rs` (like those created by `svd2rust` or `bindgen`), and splits it into structured modules. Check out the [`nrf52`] Chip Support Crate as an example, and try it out for your generated projects!
+* [astro] shared their crate [`managed`], A library that provides a way to logically own objects, whether or not heap allocation is available
```
Comment (jonas-schievink) [inline newsletters/2018-04-28.md]: I don't think @astro is the author of `managed`, only a contributor. See https://github.com/m-labs/rust-managed

Comment (jamesmunns) [reply newsletters/2018-04-28.md]: Corrected, thanks @jonas-schievink!

**Comment by japaric** [comment]
Thanks @jamesmunns!

**Pull request merged**